docs/topics/gradle.md

```diff
@@ -968,3 +968,74 @@ When configuring the Kotlin daemon's JVM arguments, note that:
   >
   {type="note"}
 * If the `Xmx` is not specified, the Kotlin daemon will inherit it from the Gradle daemon.
+
+## Defining Kotlin compiler execution strategy
+
+_Kotlin compiler execution strategy_ describes where Kotlin compiler is executed and if incremental compilation is supported in each case.
```

Comment:
Maybe `defines`? Though the current wording seems OK too.

Reply:
Defines is better indeed, thank you.